core/server/common/src/main/java/alluxio/master/journal/AbstractJournalSystem.java

```diff
@@ -22,6 +23,8 @@
   private volatile Mode mMode = Mode.SECONDARY;
   private boolean mRunning = false;
 
+  public static final AtomicBoolean ALLOW_JOURNAL_MODIFY = new AtomicBoolean(true);
```

Comment:
Should this start as false?

Reply:
it would never get set to true in non-HA mode. The only danger is that a deposed primary tries to write, so it's ok to allow modifies at the start because we aren't a deposed primary yet